Dockerfile

```diff
@@ -28,6 +28,10 @@ RUN apt-get update -qq && \
 # http://unix.stackexchange.com/questions/195975/cannot-force-remove-directory-in-docker-build
 #        && rm -rf /var/lib/apt/lists
 
+# adding custom locales to provide backward support with scrapy cloud 1.0
+RUN echo 'en_US.UTF-8 UTF-8\nde_DE.UTF-8 UTF-8\nnl_NL.UTF-8 UTF-8\nfr_FR.UTF-8 UTF-8' \
+    >> /etc/locale.gen && locale-gen
```

Comment:
It's a bit different for sure, the thing is both scrapy & portia stacks are based on different base image, so the same approach as for hworker stack doesn't work.